cmd/params/params.go

```diff
@@ -410,6 +411,7 @@ func LoadHeartbeatParams(v *viper.Viper) (Heartbeat, error) {
 		Entity:            entityExpanded,
 		ExtraHeartbeats:   extraHeartbeats,
 		EntityType:        entityType,
+		GuessLanguage:     vipertools.FirstNonEmptyBool(v, "guess-language", "settings.guess_language"),
```

Comment:
It's missing the string form of `GuessLanguage`. It should be added to `func (p Heartbeat) String() string { ... }`.